projects/neural_correlations/EXP2-HighOrder/NeuCorr_Exp2.ipynb

```diff
@@ -2,7 +2,9 @@
  "cells": [
```

Comment:
Please include a short description in each ipython notebook to describe the goal and main findings of the experiment.